src/littler.c

```diff
@@ -739,7 +739,16 @@ int main(int argc, char **argv){
         destroy_membuf(pb);
     } else if (optind < argc && (strcmp(argv[optind],"-") != 0)) {
         /* set environment variable giving path to script file */
+#if defined(HAVE_REALPATH) && defined(PATH_MAX)
+        char script_path[PATH_MAX + 1];
+        if (realpath(argv[optind], script_path) != NULL)
+            setenv("LITTLER_SCRIPT_PATH", script_path, 1);
+        else
+            setenv("LITTLER_SCRIPT_PATH", argv[optind], 1);
+#else
```

Comment:
Oh I had not realized the extra work with realpath() here but well done!  That is likely more useful.

Reply:
`realpath` is probably pretty universally available, but R checks for it so I thought I should play it safe. I'll add to the docs and changelog shortly (doing this between other stuff). Making it optional I'm not sure about - it would need a flag or an env var, I guess, neither of which feels ideal. I would hope it can be easily ignored where it's not helpful.

Reply:
I really appreciate the clean `configure.ac` work here.  

Let's drop the optional. We guess we can bear the cost of setting an env.var.